dropwizard-util/pom.xml

```diff
@@ -45,5 +45,10 @@
             <artifactId>jackson-databind</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
Please use JUnit 5 for writing tests. The dependency is already included in "test" scope for all sub-modules.

Reply:
Also assertj for assertions